src/sampling.jl

```diff
@@ -74,12 +74,23 @@ end
 
 ### draw a pair of distinct integers in [1:n]
 
+"""
+    samplepair(n)
+
+Draw a pair of integers between 1 and `n` without replacement.
```

Comment:
Oh whoops! Looks like that's something I missed going through your previous comments. Sorry about that. I'll do that in the next round. Thanks for noticing!